pkgs/development/compilers/yosys/default.nix

```diff
@@ -46,17 +43,18 @@ stdenv.mkDerivation rec {
   };
 
   enableParallelBuilding = true;
-  nativeBuildInputs = [ pkg-config protobuf flex bison python3 ];
-  buildInputs = [ tcl readline libffi python3 protobuf zlib ];
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ tcl readline libffi python3 bison flex protobuf zlib ];
 
   makeFlags = [ "ENABLE_PROTOBUF=1" "PREFIX=${placeholder "out"}"];
 
   patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
This should also have the `.patch` file moved out as well. Fixed.